pkg/volume/fc/fc_util.go

```diff
@@ -101,15 +140,28 @@ func scsiHostRescan(io ioHandler) {
 	}
 }
 
+// check scsi disk is really exist and is attached to the node using SCSI Inquiry
+func checkDiskExist(b fcDiskMounter, disk, wwid string) (bool, error) {
+	out, err := b.plugin.execCommand("/lib/udev/scsi_id", []string{"--replace-whitespace", "--page=0x83", "--whitelisted", "--device=" + disk})
```

Comment:
>>Document that scsi_id is needed. And check error message and filter scsi_id is not found error.
 
I got it. I'll add document on the fc example page.
https://github.com/kubernetes/kubernetes/tree/master/examples/volumes/fibre_channel

>>it could also be page 0x80.

I'll add the logic to check 0x80 If the 0x83 check will be error.

>>I can find /lib/udev/scsi_id on Fedora based containers, but not on latest ubuntu containers.

To be clear, this command is executed on Kubernetes host.
I think "udev" or "systemd" are essential package, therefore I suppose ubuntu or RHEL server basically have this command. Also this plugin already depends on udev because symlink under the /dev/disk/by-path is created by udev.
But I agree cmd check is necessary.

Here is my quick check results. 
Ubuntu, RHEL, Fedora have "/lib/udev/scsi_id" if udev/systemd is installed.

```
14.04LTS
https://packages.ubuntu.com/trusty/amd64/udev/filelist	/lib/udev/scsi_id

16.04LTS
https://packages.ubuntu.com/xenial/amd64/udev/filelist	/lib/udev/scsi_id

17.04
https://packages.ubuntu.com/zesty/amd64/udev/filelist	/lib/udev/scsi_id

RHEL6
$ rpm -qf /lib/udev/scsi_id 
udev-147-2.73.el6_8.2.x86_64

RHEL7
$ rpm -qf /lib/udev/scsi_id
systemd-219-30.el7_3.6.x86_64

FC25
% rpm -qf /lib/udev/scsi_id
systemd-udev-231-17.fc25.x86_64
```